### ISSUE TITLE
Index Error: "index 2 is out of bounds for axis 1 with size 2"

### ISSUE THREAD

**Issue by BHAVANAGUBBI** [ljvmiranda921/pyswarms#372]
`*` 
* Python version: 3.7.3
* Operating System: Windows

### Description

I am trying to implement the SVR model to predict the iris data set. 
But am facing the below errors.

### What I Did

```
#my SVR regressor model
from sklearn.svm import SVR
reg=SVR(kernel='linear',gamma='auto',C=1)
reg.fit(X_train[:,0:2],Y_train)

#meshgrid function
def mesh_grid(X,Y,h=0.01):
    '''
    a function that does mesh grid
    X axis --- one feature of X
    Y axis --- another feature of X
    '''
    x_min,x_max=X.min()-1,X.max()+1
    y_min,y_max=Y.min()-1,Y.max()+1
    xx,yy=np.meshgrid(np.arange(x_min,x_max,h),
                     np.arange(y_min,y_max,h))
    return xx,yy
 #plotting the model
from matplotlib.colors import ListedColormap
X_set,Y_set=X_train[:,0:2],Y_train
X1,X2=mesh_grid(X_set[:,0],X_set[:,1]) #taking out columns of sepal_length, sepal_width
plt.contourf(X1,X2,reg.predict(np.array([X1.ravel(),X2.ravel()]).T).reshape(X1.shape),
             alpha=0.75,cmap=ListedColormap(('blue','orange','green')))
for i,j in enumerate(np.unique(Y_set)):
    plt.scatter(X_set[Y_set==j,0],X_set[Y_set==j,1],X_set[Y_set==j,2],
                c=ListedColormap(('blue','orange','green'))(i),label=j)
plt.title('SVM--TRAINING SET')
plt.xlabel('Sepal Length')
plt.ylabel('Sepal Width')
plt.legend()
```
output:
Please find the error desc in the below link
<img width="413" alt="index error" src="https://user-images.githubusercontent.com/49832021/75867724-0645d300-5e2d-11ea-9b68-0ebdec9c90bd.png">

**Comment by whzup** [comment]
@BHAVANAGUBBI Welcome on pyswarms! What exactly does your example have to do with pyswarms? 🤔 It seems to me, that this is an sklearn problem.
